app/assets/javascripts/application/kids.js

```diff
@@ -0,0 +1,109 @@
+mumuki.load(function () {
+
+  mumuki.resize(function () {
+    var margin = 15;
+    var fullMargin = margin * 2;
+
+    var gbsBoard = $('[class^="mu-kids-gbs-board"]');
```

Comment:
This is wrong, we should not couple kids model to gobstones board. 

We should instead call generic classes and call generic methods. Then the runner assets should implement the proper hooks.  

For example: 

Instead of 

```
var gbsBoard = $('[class^="mu-kids-gbs-board"]');
gbsBoard.each(function (i) {
   gsBoardScale($(gbsBoard[i]));
});
```

it should be


```
var scalableComponents = $('[class^="mu-kids-scalable"]');
scalableComponents.each(function (i) {
   component = $(scalableComponent[i]);
   componen.scale();
});
```